### PR TITLE
Update SecurityPage when no Security license installed

### PR DESCRIPTION
**Pull request by kingzacko1**
## Description

Make default Graylog Security page with no license installed more informative and less like an error page. See Graylog2/graylog-plugin-enterprise#3545 for context. Open to all suggestions for changes, just getting something up for people to look at.

/nocl 

## Motivation and Context

Closes Graylog2/graylog-plugin-enterprise#3545
## How Has This Been Tested?

Purely UI change of the SecurityPage. Tested functionality of links in refactored `EnterpriseProductLink` component
## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: graylog2-web-interface/src/pages/SecurityPage.tsx [hunk 1]
```diff
@@ -16,60 +16,74 @@
...
-      {isCloud
-        ? (<>Contact your Graylog account manager.</>)
-        : (<LinkTo />)}
-    </StyledAlert>
+    <DocumentTitle title="Try Graylog Security">
+      <div>
+        <PageHeader title="Try Graylog Security">
+          {null}
+          <span>
+            Extend Graylog's capabilities for detecting, investigating, and responding to cybersecurity
```
Comment (mikedklein) [inline graylog2-web-interface/src/pages/SecurityPage.tsx:54]: Let's go with the review bot suggestion here it is technically a best practice albeit a nit.

**Review by mikedklein — CHANGES_REQUESTED** [review]
I don't think the `vendor-modul-ids.json` should be in this PR right? 

The code all looks good minus the review bot suggestion and my other suggestion.

One content suggestion is to remove the graph.

image

Since that is kind of redundant. That being said I am not entirely sure what to put there.

**Comment by kingzacko1** [comment]
@mikedklein addressed your feedback and cleaned up bot suggestion. I'm open to any ideas about what else to put on that page instead of the traffic stuff.

**Comment by kingzacko1** [comment]
closing. This was handled in #15300

**Pull request closed without merging**